reactComponents/src/searchbar/tokenSearch/ResultDetail.tsx

```diff
@@ -254,9 +254,12 @@ export const ResultDetail: FC<DetailType> = (props: DetailType) => {
 
   const isActive = index === currentIndex;
   const tokenImage = (token) => {
-    if (token?.image)
-      return <img alt={token?.symbol} src={token?.image} style={{ borderRadius: '50%' }} width={imageSize} />;
-    else return <DefaultIcon />;
+    console.log(TokensList[token?.address?.toLowerCase()])
+    const tokenImageUrl = token?.image || TokensList[token?.address?.toLowerCase()];
```

Comment:
can you turn this into a getter function?

It also looks like a lot of that boilerplate can be removed by a dynamic string
```
`https://raw.githubusercontent.com/ava-labs/bridge-tokens/main/avalanche-tokens/${address}/logo.png`
```

Having all that as you have it will mean a worse mobile experience. The app is already 10mb to download. We should keep it light where possible